addons/mail/static/src/components/message/message.scss

```diff
@@ -243,6 +243,11 @@
     &.o-has-message-selection:not(.o-selected) {
         opacity: 0.5;
     }
+
+    &.o-highlighted {
+        background-color: rgba($o-brand-primary, .1);
+        border-left: 4px solid $o-brand-primary;
```

Comment:
Using a border is problematic because it is offsetting the rest of the message, it gets missaligned compared to other messages.

Either other messages need an "empty" border of the same width to compensate, or it must be done differently.

Reply:
> Using a border is problematic because it is offsetting the rest of the message, it gets missaligned compared to other messages.
> 
> Either other messages need an "empty" border of the same width to compensate, or it must be done differently.

Done!!